### ISSUE TITLE
Problem in the code with 10-fold validation

### ISSUE THREAD

**Issue by FAhtisham** [Marscolono/Adapt-Kcr#2]
Hi,

I have tried reproducing your results on 10-fold cross-validation and your cross-validation loop is incorrect (your training script).

	       `train_loader = Data.DataLoader(Data.TensorDataset(X_train,y_train), BATCH_SIZE, shuffle = False)
		val_loader = Data.DataLoader(Data.TensorDataset(X_valid, y_valid), BATCH_SIZE, shuffle = False)

		model = net`

Here, when you are setting `model=net` then python will create a pointer referring to the original model which has been trained on the rest of the data which might come under the validation fold as well.

therefore, I believe the numbers reported in the paper are also incorrect and they should be around 83-84% in terms of cross-validation. This is also evident from your performance on the independent test set  (85.2)

I hope so you will resolve this issue in the repo and also upload the correct numbers.
